src/Avalonia.Visuals/Media/FontFamily.cs

```diff
@@ -79,9 +79,23 @@ internal FamilyNameCollection FamilyNames
         /// </returns>
         public override string ToString()
         {
+            if (Key != null)
+            {
+                return Key + "#" + Name;
+            }
+
             return Name;
         }
 
+        /// <summary>
+        /// Implicit conversion of FontFamily to string
+        /// </summary>
+        /// <param name="fontFamily"></param>
+        public static implicit operator string(FontFamily fontFamily)
```

Comment:
I think this should have been the other way around, i.e. string to FontFamily.

Reply:
The FontFamily was defined as a string so if someone still has some code with FontFamily = "MyFont" it still works. The other way isn't really needed. There is already ToString